spec/requests/users_controller_spec.rb

```diff
@@ -253,7 +253,7 @@
 
       User.roles.reject { |role| role == "trial" }.each_value do |role_value|
```

Comment:
Please make this consistent with the other file where we are more explicit with our specs.

Reply:
This has been changed so that we aren't looping through the roles, instead we're testing specific roles and actions in each test.